src/pages/element/link-editor.jsx

```diff
@@ -7,6 +7,9 @@ var api = require('../../lib/api');
 var platform = require('../../lib/platform');
 var types = require('../../components/basic-element/basic-element.jsx').types;
 
+// var backgroundColors = ['#9FD0E0', '#99CA47', '#EFC246', '#E06A2C', '#69A0FC', '#8173E4'];
```

Comment:
Wasn't sure if it was going to come back in once #719 was discussed further. I can remove it, but I was leaving it for some future reference.